tensorflow/python/eager/function.py

```diff
@@ -1535,6 +1535,10 @@ def __init__(self,
               inputs=self.inputs,
               captured=self._captured_inputs,
               closures=self._captured_closures))
+
+    self._captured_inputs += nest.flatten(
```

Comment:
Does it have any implications on other places that uses `self._captured_inputs` (if there is any)?

Reply:
Doesn't look like it

Reply:
I think the failing tests, "testDeferredCapture*", in the other comment could be related,

Reply:
Confirmed, it turns out that a certain return value cannot be cached. See comment at end of discussion